glide.yaml

```diff
@@ -184,7 +184,7 @@ import:
   version: b64f5908f4945f4b11ed4a0a9d3cc1e23350866d
 # keep us up to date with latest DNS security fixes
 - package: github.com/miekg/dns
-  version: ~v1
+  version: 5a2b9fab83ff0f8bfc99684bd5f43a37abe560f1
```

Comment:
I had to pin it to this specific version because the newer version requires `go 1.10+` and I think that `3.10` is pinned to `go 1.9`